details/encryption.md

```diff
@@ -242,7 +242,7 @@ like the helper party origin, is expected to be implicit, so it should not
 require transmission.
 
 For a 64-bit match key, this means that it is possible to represent each
```

Comment:
Are we looking at 64 bit match keys or 40 bit? Do you think moving to 40bit match keys will reduce the number of bytes further?

Reply:
I honestly don't know.  40 bits means a LOT higher risk of collision.  You only need to start to get close to a million people before you start to get non-negligible collision probability.  It saves a few bytes (16->10), but it is the saving in terms of the sort that I think would drive us toward a smaller value.  And for that, we can also do what Ben Case suggested to reduce a larger space with a PRF.

Collision risk is therefore largely a utility problem, so we should probably defer to the people who are paying.  The protocol and implementation can afford to be flexible to some extent until then.